recipes/pyode/build.sh

```diff
@@ -0,0 +1,23 @@
+# stage 1, library
+mkdir _build
+cd _build
+
+cmake -DCMAKE_BUILD_TYPE=Release \
+      -DODE_WITH_DEMOS:BOOL=OFF \
+      -DODE_WITH_TESTS:BOOL=OFF .. \
+      -DCMAKE_INSTALL_PREFIX:PATH="" \
+      -DCMAKE_INSTALL_LIBDIR="lib"
+
+make VERBOSE=1
+make install DESTDIR="${PREFIX}"
+cd ..
+
+# stage 2, bindings
+
+# make sure recently installed ode.pc is found
+PKG_CONFIG_PATH=${PREFIX}/lib/pkgconfig:
```

Comment:
Is this needed? Naively, I expect it to not to do anything

Reply:
Indeed, not needed.